app/src/main/java/org/eyeseetea/malariacare/domain/exception/NetworkException.java

```diff
@@ -0,0 +1,8 @@
+package org.eyeseetea.malariacare.domain.exception;
+
+
+public class NetworkException extends Exception {
+    public NetworkException() {
+        super("Credentials not valid");
```

Comment:
Sorry @xurxodev, I explainned wrong,  i was talking about the message, "Credentials not valid" is not the correct message for networkException